packages/mermaid/src/dagre-wrapper/clusters.js

```diff
@@ -63,17 +64,18 @@ const rect = (parent, node) => {
     .attr('width', width)
     .attr('height', node.height + padding);
 
+  const { subGraphTitleTopMargin } = getSubGraphTitleMargins(getConfig());
```

Comment:
Done, I named the const `siteConfig`, is that an okay name? I avoided something generic like `config` since it already exists in the js default library.